frontend/src/server/db/schema.ts

```diff
@@ -0,0 +1,97 @@
+// Example model schema from the Drizzle docs
```

Comment:
Similarly, this appears to be server-side table creation. Can probably delete this now that you're using the `fetchProfessors()` and `fetchCourses()` functions.